pkg/api/v1/resource/helpers_test.go

```diff
@@ -720,7 +720,7 @@ func TestPodResourceRequests(t *testing.T) {
 		{
 			description: "resized, infeasible",
 			expectedRequests: v1.ResourceList{
-				v1.ResourceCPU: resource.MustParse("2"),
```

Comment:
Why are we changing these resource size in tests?🤔️

Reply:
According to the chapter `Version Skew Strategy` of the KEP-1287, when  kube-scheduler computes the pod resource requests,  it should use max(AllocatedResources, Requests) if PodStatus.Resize != "".

In this test case, the PodStatus.Resize is equal to Infeasible and the AllocatedResources is less than Requests, so the expected result is Requests. I'm not sure that this change is reasonable when the PodStatus.Resize is equal to Infeasible. If I misunderstood, please correct me. Thanks.

FYI:

- https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#version-skew-strategy
- https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L2576

Reply:
Ah I see, sgtm